charts/trustify/templates/helpers/_infrastructure.tpl

```diff
@@ -38,6 +38,8 @@ Arguments (dict):
 {{- if eq ( include "trustification.application.tracing.enabled" . ) "true" }}
 - name: TRACING
   value: "enabled"
+- name: METRICS
```

Comment:
Shouldn't this be covered by a check for `metrics.enabled`? Otherwise, it looks like the `metrics.enabled` flag doesn't get used.

Reply:
thanks I'll take a look :+1:

Reply:
PR updated